composer.json

```diff
@@ -16,20 +16,28 @@
         }
     ],
     "require": {
-       "php": "^7.1.0",
+        "php": "^7.1.0",
         "symfony/security": "^3.0 || ^4.0",
         "symfony/config":"^3.0 || ^4.0",
         "symfony/http-foundation": "^3.0 || ^4.0",
         "symfony/dependency-injection": "^3.0 || ^4.0",
-        "symfony/http-kernel": "^3.0 || ^4.0"
+        "symfony/http-kernel": "^3.0 || ^4.0",
+        "symfony/monolog-bundle": "^2.8 || ^3.0"
     },
     "require-dev": {
         "phpspec/phpspec": "^2.5|^3.0",
         "phpstan/phpstan": "^0.8.0"
```

Comment:
This comment is not directly related to this PR but why do we have this requirement in dev?

Reply:
I think I add it with #23, think we can now remove it